intro.txt

```diff
@@ -127,22 +127,35 @@ the \texttt{libest} directory, run the configure script and then make:
 
 This package has been tested on both 32 and 64 bit Linux systems running
 CentOS and Ubuntu.  The OpenSSL development package should be installed
-on the system.  libest is designed to work 
+on the system, or OpenSSL can be used.  libEST is designed to work 
```

Comment:
This paragraph is messed up.  It appears that a mechanical change was made to remove references to CiscoSSL, but in doing so, the context of the sentences gets messed up both OpenSSL and CiscoSSL were likely being referenced.